src/version.cpp.in

```diff
@@ -0,0 +1,9 @@
+#include <string>
+
+#include <bbp/sonata/common.h>
+
+namespace bbp {
+namespace sonata {
+const std::string VERSION("@SONATA_VERSION@");
```

Comment:
Just in case: beware of https://github.com/BlueBrain/MorphIO/pull/32-like trap, should you decide to use this constant for some other static initialization.

Reply:
Good catch. I'll change it to a function then.